content/en/blog/releases/changes-24-august-2018.md

```diff
@@ -15,8 +15,8 @@ Before we go over whats happened - a big thanks to all of our new committers and
 
 Here's a quick summary of some of the main things that's happened this summer:
 
-* we have awesome [new tutorials](/about/tutorials/) to get you started even faster on GKE
-* we now have [full support AWS both kops and EKS](/getting-started/create-cluster/#using-amazon-aws) with ECR support OOTB thanks to the excellent eksctl tool which helps us create EKS clusters on Amazon
+* we have awesome [new tutorials](https://jenkins-x.io/tutorials/video/) to get you started even faster on GKE
```

Comment:
for this we should probably just link to /getting-started/demos-talks-posts/create_cluster_gke/ ?